File: 🟢 Easy/9 - Palindrome Number/_9PalindromeNumber.java


class Solution {
    public boolean isPalindrome(int x) {
        int num = x;
        int rev = 0;
        while (x > 0) {
            int r = x % 10;
            rev = rev * 10 + r;
            x = x / 10;
        }
        if (num == rev)
            return true;
        return false;
    }
}

File: 🟢 Easy/9 - Palindrome Number/_9PalindromeNumber.js
var isPalindrome = function (x) {
    let num = x;
    let rev = 0;
    while (x > 0) {
        let r = x % 10;
        rev = rev * 10 + r;
        x = Math.floor(x / 10);
    }

    return num == rev;
};